src/styles/modal.css

```
@@ -82,7 +83,151 @@ body.modal-open {
+hr {
+  width: 100%;
+}
+
+/* 모바일 */
+@media (max-width: 768px) {
+  .modal {
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+
+    top: auto;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)